apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/TestProtos/geometry.proto

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.cloud.spanner.data.common_testing;
+
+message Point {
+  double x = 1;
+  double y = 2;
+}
+
+message Rectangle {
+  Point top_right = 1;
```

Comment:
Not that it's really relevant, but top *right* seems like an odd way to define a rectangle. (I'd always expect top left + width and height, or top/left/bottom/right coordinates.)

Reply:
(No need to change though.)

Reply:
I'm pretty sure I was thinking of `top_left` because that's what feels natural to me as well.

I'll leave it as is now but I'll probably change it when we do a pass to unskip the tests.